components/hab/install.sh

```diff
@@ -285,13 +285,38 @@ do_install() {
 # Download location for the temporary files
 tmp_dir="${TMPDIR:-/tmp}/hab"
 
+# use stable channel by default
+channel="stable"
+
+# ## CLI Argument Parsing
+
+# Parse command line flags and options.
+while getopts "c:v:" opt; do
+  case $opt in
+    c)
+      channel=$OPTARG
+      ;;
+    v)
+      version=$OPTARG
+      ;;
+    \?)
+      print_help
+      exit_with "Invalid option: -$OPTARG" 1
```

Comment:
We don't have the `exit_with()` function here, but could replace that line with `echo ">>> Invalid option: -$OPTARG"; exit 1` to do an equivalent thing.